lib/vector-core/src/event/log_event.rs

```diff
@@ -531,6 +540,7 @@ mod test {
     }
 
     #[test]
+    #[allow(clippy::too_many_lines)]
```

Comment:
I could see disabling this lint globally. I've never found line count lints particular useful.

Reply:
I like it personally as a soft backstop. I'm open to disabling it in a future PR if we keep hitting it.